integration/users/usersinfo_test.go

```diff
@@ -76,11 +77,27 @@ func TestUsersinfo(t *testing.T) {
 				createUser("singleuser", "123456"),
 			},
 		},
+		{
+			dbSuffix: "nomongo",
+			payloads: []bson.D{
+				{
+					{"createUser", "WithPLAIN"},
+					{"roles", bson.A{}},
+					{"pwd", "pwd1"},
+					{"mechanisms", bson.A{"PLAIN"}},
+				},
+			},
+			skipForMongoDB: true,
+		},
 	}
 
 	dbPrefix := testutil.DatabaseName(t)
 
 	for _, inserted := range dbToUsers {
+		if inserted.skipForMongoDB && setup.IsMongoDB(t) {
```

Comment:
That should be a subtest with expected failure. Otherwise, it just too easy to silently skip it

Reply:
The reason why I haven't added it as a subtest is that this is for setting up the environment for multiple tests, rather than for testing something. I want it to run before any subtests (that are executed as a second step).